GOVERNANCE.md

```diff
@@ -181,24 +181,54 @@ yourself from the MAINTAINERS file.
-wishes to remain a maintainer on the project, they may convert their role to
-a reviewer until they are able to perform committer duties again.
-
-If an inactive maintainer wants to remain in their current role or does not open
-their own pull request to change roles, another maintainer may open up a pull
-request to the MAINTAINERS file to change their role. Role changes and removals
-must be approved by 2/3 of the current committers. The voting and discussion
-period is seven days, after which a committer will verify the votes,
+be removed from the list if they do not show consistent activity on the
+project. Depending on the existing role, committer or reviewer, there are
+variations to the removal process detailed as follows.
+
+### Committer removal
+
+For committers, the maintainers periodically review committer project
+activity. The maintainers use the CNCF "devstats" community metrics tool for
+data on user activity statistics.
+
+If a committer has shown no activity over a six-month period, the committer
+will be contacted to notify them of their proposed Emeritus role. There is
+no automatic change of status in the project. First, the activity status is
+discussed directly with the committer, and second, maintainers discuss whether
+other non-tracked contributions to the project reflect an ongoing, active
+participation in the project.
+
+Based on the outcome of these discussions, the committer may choose to remove
+themselves from the MAINTAINERS file, or another maintainer may open a
+pull request to the MAINTAINERS file on their behalf for removal.
+Removals must be approved by 2/3 of the current committers. The voting
+and discussion period is seven days, after which a committer will verify the votes,
+merge the pull request, and apply the changes to the organization.
+
+### Emeritus maintainers
+
+For committers who are stepping down, the project would like to memorialize their
+contributions to the project by recognizing them as Emeritus maintainers in
+the EMERITUS.md file. The EMERITUS.md file will include a brief paragraph
+summarizing their contribution to the containerd project and recognize them as
+permanent Emeritus members of the community.
+
+If in the future an Emeritus maintainer has the desire or ability to return to
+contributing to the project, Emeritus maintainers can submit a pull request
+reversing their removal from the MAINTAINERS file and approval only requires
+2 LGTMs from existing committers to return to full committer status in the
+project.
+
+### Reviewer removal
+
+While reviewers are not voting members of the containerd project, the maintainers
+may assess reviewer activity periodically and recommend removals for any
+completely inactive reviewers.
+
+Reviewers may remove themselves from the MAINTAINERS file if they wish to or
+any existing maintainer may open a pull request to the MAINTAINERS file on their
+behalf. Removals must be approved by 2/3 of the current committers. The voting
```

Comment:
Does this merit a lower bar?

Reply:
Adding reviewers only requires 1/3.  Maybe we move this to a simple majority and clarify it's not needed for stepping down?

```suggestion
behalf. Non-voluntary removals must be approved by a simple majority of the current committers. The voting
```

Reply:
I think after we've moved the existing ex-maintainers who "stepped down" to reviewer, then yes perhaps 1/3 or simple majority..  Suggest leaving it at 2/3 for now for that reason.